lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:askm/core/context_extensions.dart';
+import 'package:askm/generated/assets/assets.gen.dart';
+import 'package:askm/presentation/tokens/spacing.dart';
+import 'package:askm/presentation/widgets/background_widget.dart';
+import 'package:askm/presentation/widgets/custom_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SocialSignUpLayout extends StatefulWidget {
+  const SocialSignUpLayout({super.key});
+
+  @override
+  State<SocialSignUpLayout> createState() => _SocialSignUpLayoutState();
+}
+
+class _SocialSignUpLayoutState extends State<SocialSignUpLayout> {
+  static const upperFlex = 5;
+  final appLogo = SvgPicture.asset(Assets.images.logo);
```

Comment:
You're cluttering up the layout with such things, there's no need for them in the layout, it's better to store them in the widget